<ui version="4.0" >
 <class>SpectralMeterConfigWidget</class>
 <widget class="QWidget" name="SpectralMeterConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FFT-Spectrum Configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Frequency Range</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="5" column="0" colspan="2" >
       <widget class="QCheckBox" name="checkBoxAverage" >
        <property name="text" >
         <string>Show average spectrum</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSpinBox" name="spinBoxNumBands" >
        <property name="maximum" >
         <number>2048</number>
        </property>
        <property name="minimum" >
         <number>4</number>
        </property>
        <property name="value" >
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Number of bands:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="spinBoxLowerDb" >
        <property name="maximum" >
         <number>0</number>
        </property>
        <property name="minimum" >
         <number>-140</number>
        </property>
        <property name="value" >
         <number>-140</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Lower dB value:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spinBoxUpperDb" >
        <property name="maximum" >
         <number>12</number>
        </property>
        <property name="minimum" >
         <number>-24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Upper dB value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinBoxLowerFreq" >
        <property name="suffix" >
         <string> Hz</string>
        </property>
        <property name="maximum" >
         <number>20000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Lower Limit:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinBoxUpperFreq" >
        <property name="suffix" >
         <string> Hz</string>
        </property>
        <property name="maximum" >
         <number>96000</number>
        </property>
        <property name="minimum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Upper Limit:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAdvanced" >
     <property name="title" >
      <string>Advanced FFT Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>FFT Size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="comboBoxFftSize" >
        <property name="currentIndex" >
         <number>3</number>
        </property>
        <item>
         <property name="text" >
          <string>256</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>512</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>1024</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>2048</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>4096</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>8192</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="comboBoxWindowing" >
        <item>
         <property name="text" >
          <string>Rectangle</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Hanning</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Hamming</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Blackman</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Windowing function:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonAdvanced" >
       <property name="text" >
        <string>Advanced</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply" >
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxUpperFreq</tabstop>
  <tabstop>spinBoxLowerFreq</tabstop>
  <tabstop>spinBoxUpperDb</tabstop>
  <tabstop>spinBoxLowerDb</tabstop>
  <tabstop>spinBoxNumBands</tabstop>
  <tabstop>checkBoxAverage</tabstop>
  <tabstop>comboBoxFftSize</tabstop>
  <tabstop>comboBoxWindowing</tabstop>
  <tabstop>buttonApply</tabstop>
  <tabstop>buttonClose</tabstop>
  <tabstop>buttonAdvanced</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAverage</sender>
   <signal>clicked()</signal>
   <receiver>buttonApply</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>29</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel" >
     <x>270</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
